draft-ietf-rats-architecture.md

```diff
@@ -424,17 +424,19 @@ of the TPM together with whatever component is feeding it the measurements.
 
 ## Layered Attestation Environments {#layered-attestation}
 
-By definition, the Attester role generates Evidence.
-An Attester may consist of one or
-more nested environments (layers).
-The root layer of an Attester includes at least one root of trust.
-In order to appraise Evidence generated by an Attester, the Verifier needs to trust the Attester's root of trust.
-Trust in the Attester's root of trust can be established either directly (e.g., the Verifier puts the root of trust's public key into its trust anchor store) or transitively via an Endorser (e.g., the Verifier puts the Endorser's public key into its trust anchor store).
-In layered attestation, a root of trust is the initial Attesting Environment.
-Claims can be collected from or about each layer.
-The corresponding Claims can be structured in a nested fashion that reflects the nesting of the Attester's layers.
-Normally, Claims are not self-asserted, rather a previous layer acts as the Attesting Environment for the next layer.
-Claims about a root of trust typically are asserted by Endorsers.
+For layered attestation there is a root Attester and several other attesters that are not roots.
+The root attester is also called the initial Attester.
+
+The Verifier comes to trust the root Attester in any of the normal ways, for example, through an Endorsement.
+Trust in the Attesters layered above are not established in any of the normal ways.
+Instead, the root attester provides Evidence about the other attester layers. 
+
+That is, the root Attester collects and provides claims about Attester in the layer immediately above.
```

Comment:
This implies that there are multiple Attesters, (one per layer), and this is change, because the original text had one Attester collecting Claims from multiple layers.

This change would have implications to many other documents, TEEP, EAT, and CCC white paper and other sections of this document.